FILES.deploy-tools/root/inventory/HA

```diff
@@ -0,0 +1,74 @@
+#
+# HA production cluster with all the trimmings
+# 
+#
+[all:vars]
+logical_cluster_name=ha
+cluster_name=cluster.local
+native_dns=os.ncsa.edu
+image=CoreOSAlpha
+flavor=r2.medium
+kube_version=1.2.4
+kube-ui=true                                          
+kube-dash=true
+key_name={{logical_cluster_name}}
+cluster_fs_hostpath=/media/glfs 
+
+[compute:vars]
+flavor=n-rc1.large
+
+[glfs:vars]
+vol_size=200
+vol_name=brick0
+brick_mount_path=/media/brick0
+service_name=media-brick0.mount
+
+#
+# Some optional things
+# 
+cluster_monitoring=true
+cluster_logging=true
+kube-dash=true
+kube-ui=true
```

Comment:
kube_ui and kube_dash are also defined in the top section

Is this intentional?